src/components/visualizations/chart/highcharts/test/getAxisNameConfiguration.spec.ts

```diff
@@ -56,4 +58,90 @@ describe("getAxisNameConfiguration", () => {
             ],
         });
     });
+
+    it.each([
+        [
+            `${VisualizationTypes.COMBO} chart`,
+            {
+                type: VisualizationTypes.COMBO,
+                xAxes: [{}, { opposite: true }],
+                yAxes: [{}, { opposite: true }],
+            },
+        ],
+        [
+            `${VisualizationTypes.COMBO2} chart`,
+            {
+                type: VisualizationTypes.COMBO2,
+                xAxes: [{}, { opposite: true }],
+                yAxes: [{}, { opposite: true }],
+            },
+        ],
+        [
+            `${VisualizationTypes.COLUMN} chart`,
+            {
+                type: VisualizationTypes.COLUMN,
+                xAxes: [{}, { opposite: true }],
+                yAxes: [{}, { opposite: true }],
+            },
+        ],
+        [
+            `${VisualizationTypes.LINE} chart`,
```

Comment:
Please move "chart" word to name of the test and you can remove string composition. There is no reason for it.